ignite/pkg/cosmosgen/generate_javascript.go

```diff
@@ -60,10 +61,27 @@ func (g *jsGenerator) generateModules() error {
 
 	gg := &errgroup.Group{}
 
+	dirCache := cache.New[[]byte](g.g.cacheStorage, "generate.javascript.dirchange")
 	add := func(sourcePath string, modules []module.Module) {
 		for _, m := range modules {
 			m := m
-			gg.Go(func() error { return g.generateModule(g.g.ctx, tsprotoPluginPath, sourcePath, m) })
+			gg.Go(func() error {
+				cacheKey := m.Pkg.Path
```

Comment:
cc @bjaanes

Reply:
Oh, maybe I misunderstand this? I added the includeDirs to the paths for the checksum, but should all the paths also be part of the cachekey you think? They way it is now, if the includeDirs have changed it will trigger a rebuild.

Reply:
Hey, I added more explanation here: https://github.com/ignite-hq/cli/issues/2511

Reply:
I think this can be addressed another time.